packages/angular/src/generators/scam-pipe/schema.d.ts

```diff
@@ -0,0 +1,9 @@
+export interface Schema {
+  name: string;
+  path?: string;
+  project?: string;
+  skipTests?: boolean;
+  inlineScam?: boolean;
+  flat?: boolean;
+  prefix?: string;
```

Comment:
this is not a known argument for Pipes:
https://github.com/angular/angular\-cli/blob/master/packages/schematics/angular/pipe/schema.json